models/DataObject/Data/Hotspotimage.php

```diff
@@ -181,6 +181,19 @@ public function getThumbnail($thumbnailName = null, $deferred = true)
         }
 
         if ($crop) {
+            if ($thumbConfig->hasMedias()) {
+                $medias = $thumbConfig->getMedias() ?: [];
+
+                foreach ($medias as $mediaName => $mediaConfig) {
+                    $thumbConfig->addItemAt(0, 'cropPercent', [
```

Comment:
Shouldn't be this `$mediaConfig` instead of ` $thumbConfig`?

Reply:
Should be fine ```addItemAt``` handles this correctly with the last param

Reply:
Ah, thanks, have overssen the last param, thanks for clarifying!